### ISSUE TITLE
Can't import more than one collection in a non-enterprise tier.  

### ISSUE THREAD

**Issue by michelc1** [bitwarden/browser#868]
A non-enterprise account can only have 2 collections but when trying to import a Bitwarden json file, with two collections (an empty organization at this point), it throws an error "This organization can only have a maximum of 2 collections". 

I suspect the `Unassigned` collection is to blame here? Importing one collection works as expected. 

**Steps to reproduce:**

1. Create two collections.
2. Add an item to each collection.
3. Export the collection.
4. Delete both the collections that you made previously. 
5. Try to import the export you performed in step 3.

**Comment by kspearrin** [comment]
When you first create your org, there is a "Default Collection" created. You would need to delete that.

**Comment by michelc1** [comment]
Sorry but I don't have a "Default Collection", am I missing something here?
>
![screen shot 2019-02-13 at 4 13 39 pm](https://user-images.githubusercontent.com/9553607/52744329-6d86b000-2faa-11e9-91e6-b918d784fec6.png)
![screen shot 2019-02-13 at 4 14 32 pm](https://user-images.githubusercontent.com/9553607/52744350-7ecfbc80-2faa-11e9-83b4-a3c53c325f83.png)

**Comment by kspearrin** [comment]
Please show an example import file you are trying to use. The only explanation would be that your import is trying to create more than two collections.

**Comment by michelc1** [comment]
Here is an example json file I am using to import items. Note, this is the same file I get from exporting via the WebUI (though I removed the organization id).

Logging out and back in, doesn't fix the problem. Have you been able to reproduce?

```
{
  "collections": [
    {
      "id": "25fb4f6e-a2f2-42f6-aa0c-a9f3015fa99c",
      "organizationId": "removed",
      "name": "Test1"
    },
    {
      "id": "4697614a-976a-4a99-8bd2-a9f40129bc08",
      "organizationId": "removed",
      "name": "Test2"
    }
  ],
  "items": [
    {
      "id": "68774f69-71dd-40c7-925f-a9f301550cd8",
      "organizationId": "removed",
      "folderId": null,
      "type": 2,
      "name": "Test1",
      "notes": "Test1",
      "favorite": false,
      "secureNote": {
        "type": 0
      },
      "collectionIds": [
        "25fb4f6e-a2f2-42f6-aa0c-a9f3015fa99c"
      ]
    },
    {
      "id": "a1617b3f-e096-4e38-8dfb-a9f301551a59",
      "organizationId": "removed",
      "folderId": null,
      "type": 2,
      "name": "Test2",
      "notes": "Test2",
      "favorite": false,
      "secureNote": {
        "type": 0
      },
      "collectionIds": [
        "4697614a-976a-4a99-8bd2-a9f40129bc08"
      ]
    }
  ]
}
```

**Comment by kspearrin** [comment]
If you exported the file, then I am guessing you already have these collections in your vault? Or did you delete them after exporting?

**Comment by michelc1** [comment]
I deleted them right after export, logged out and back in.

**Comment by kspearrin** [comment]
Thanks. Should be fixed in next version with https://github.com/bitwarden/server/commit/558cf87e17228a76a034171e78c4f1727392a4b8

**Comment by sfrooster** [comment]
There is some confusion remaining here. I'm using both the macOs desktop app

Version 2024.7.1
Shell 31.2.0
Renderer 126.0.6478.127
Node 20.15.0
Architecture arm64

and the web app in Chrome (Version 127.0.6533.120 (Official Build) (arm64)).

I have a free account with two vaults, one of which is an organization, call it O. I've added one collection to O, call it C1. In both the desktop and web apps (in password manager view) organization O shows a single collection, C1.

I added a second collection, call it C2, to O using the admin console.

I have a login/password export file in CSV format that I've tried to import using both the desktop and web apps into collection C2, and both complain that I can have a maximum of 2 collections.

To answer the question unasked - yes, in the admin console (and only there) under Collections for organization O there is a collection named Unassigned.

One more data point - I did also try importing the file into organization O, specifying collection C2 when C2 had not been created - this also didn't work.

Here is my confusion. I've seen "unassigned" referred to as a filter and it's not clear if that "unassigned" is the same or different than the "Unassigned" which shows in the admin console collections. I think they are different but the fact that Unassigned doesn't appear under Collections in the Password Manager view gives me pause.

As a final note, the "Unassigned" collection was created by the system and I'm unable to delete it (I'm sure there's a good reason for this), which I'd expect to exclude it from the two collection maximum count.

**Comment by eliykat** [comment]
Hi @sfrooster, the "Unassigned" collection is more like a filter for items that do not belong to a collection - it doesn't count towards the 2 collection limit.

I suspect your import is trying to create new collections for the imported items, rather than importing them into your existing collections. You can import into existing collections by:
* selecting an existing collection on the import page, or
* using the [json format](https://bitwarden.com/help/condition-bitwarden-import/#import-to-existing-collections) and specifying the collection IDs

See also [the FAQ](https://bitwarden.com/help/import-faqs/#q-how-do-i-import-items-directly-to-collections).

It's been a while since I've done any work on our importers, if you're still having trouble with it please contact our support team and they'll be able to help you further: https://bitwarden.com/contact/

**Comment by sfrooster** [comment]
Thanks for your reply, Thomas. But, as I indicated in my note, I did try it
the way you suggested using both the desktop (macOs, very recent) app and
the web app with chrome.

I ultimately imported them a different way which means I need to make some
changes to my duplicate identification code but no big deal.

If I get a chance to look further into this and find the problem, I'll post
it.

-Michael
